src/components/UserRegister.tsx

```diff
@@ -4,10 +4,10 @@ import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
 import LoadingOverlay from 'react-loading-overlay-ts';
 import Grid from '@material-ui/core/Grid';
 import TextField from '@material-ui/core/TextField';
-import Australia from 'flags/Australia.png';
-import america from 'flags/america.png';
+import Australia from 'assets/Australia.png';
+import america from 'assets/america.png';
 import FlagSelect from 'components/FlagSelect';
-import { IFormInput } from 'components/interfaces';
+import { IFormInput } from 'config/interfaces';
```

Comment:
interfaces shouldn't be put in config folder!!
we will be putting app configurations in it